docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -145,82 +352,12 @@ This request lists services in the apimlId domain.
-
-**Reason**  
-The trust between the domain and the Cloud Gateway was not established. 
-
-**Action**  
-Review your certificate configuration.
-
-### No debug messages similar to Gateway-CA32 completed with onComplete are produced
-
- **Reason**  
- Domain Gateway is not correctly onboarded to Discovery Service in Central API ML. 
- 
- **Action**  
- Review Gateway static definition. Check the Central Discovery Service dashboard if the domain Gateway is displayed. 
-
-## Onboarding a domain cloud-gateway service to central discovery service
-
-The central Cloud Gateway can onboard Cloud Gateways of all domains. This service onboarding can be achieved similar to additional registrations of the Gateway. This section describes the dynamic configuration of the yaml file and environment variables, and how to validate successful configuration.
-
-- Dynamic configuration via zowe.yaml
-- Dynamic configuration via Environment variables
-
-### Dynamic Configurations to the central Discovery Service
-
-#### Dynamic configuration: YML
-
-Users must set the following property for the domain cloud-gateway to dynamically onboard to the central Discovery Service.
-
-`components.cloud-gateway.apiml.service.additionalRegistration`
-
-Use the following example as a template for how to set the value of this property in zowe.yml.
-
-**Example:**
-```
-components.cloud-gateway.apiml.service.additionalRegistration:
-    # central API ML (in HA, for non-HA mode use only 1 hostname)
-       - discoveryServiceUrls:      https://ca32.lvn.broadcom.net:27554/eureka/,https://ca32.lvn.broadcom.net:37554/eureka/
- 	    routes:
-              - gatewayUrl: /
-                serviceUrl: /
-```
-
-#### Dynamic configuration: Environment variables
-
-The list of additional registrations is extracted from environment variables. You can define a list of objects by following YML->Environment translation rules. 
-
-The previous example can be substituted with the following variables:
-
-```
-ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_0_DISCOVERYSERVICEURLS=https://ca32.lvn.broadcom.net:27554/eureka/,https://ca32.lvn.broadcom.net:37554/eureka/
-ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_0_ROUTES_0_GATEWAYURL=/
-ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_0_ROUTES_0_SERVICEURL=/
-```
-
-This Zowe configuration transforms the zowe.yaml configuration file into the environment variables that are shown above. 
-
-### Validating successful configuration
-
-The corresponding ‘Cloud-Gateway’ service should appear in the Eureka console of the central discovery service. 
-
-To see details of all instances of the ‘CLOUD-GATEWAY’ application, perform a **GET** call on the following endpoint of the central discovery service:
-
-```
-/eureka/apps
-```
- 
 ## Gateway static definition example
+
```

Comment:
I know this is not part of this particular pull request, but could we please define what is specifically meant by "This file"?

Reply:
I think we also need to mark the section as deprecated.

Reply:
I agree. This looks good.